calour/heatmap/heatmap.py

```diff
@@ -319,7 +326,7 @@ def heatmap(exp: Experiment, sample_field=None, feature_field=None,
         except KeyError:
             raise ValueError('Feature field %r not in feature metadata' % feature_field)
         # numcols instead of numrows because it is transposed
-        _set_axis_ticks(ax, 'y', ticklabels, yticks_max, numcols, yticklabel_kwargs, yticklabel_len)
+        _set_axis_ticks(ax, 'y', ticklabels, yticks_max, numcols, yticklabel_kwargs, yticklabel_len, transition=False)
```

Comment:
What does it mean we send transition=False? Is there a way to override?

Reply:
not in this function's API. I feel like in our use cases, there is no need to collapse/group y tick label and white horizontal line is distracting. so I don't want to add another function parameter here.

Reply:
i used the y axis bars a few times when grouping by taxonomy and labeling with taxonomy names.
I think better to keep an option (and it's also symmetric with the x-axis, so does not complicate things...)?
can discuss tomorrow